docs/source/technical_tutorials/working_with_r/gedi_access_r.ipynb

```diff
@@ -0,0 +1,906 @@
+{
```

Comment:
<p>Move this after the search. It's best practice to get credentials right before you need a file since it has a 1 hours expiration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>